Competition2017/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -170,6 +173,24 @@ public void setupDriveCommand(
     }
     
     @Inject
+    public void setupVisionCommands(
+            OperatorInterface oi,
+            RotateRobotToBoilerCommand rotateCommand,
+            DriveToBoilerWithVisionCommandGroup driveToBoilerCommand,
+            SetRobotHeadingCommand setHeadingCommand,
+            RotateToHeadingCommand rotateToHeadingCommand
+    )   {
```

Comment:
At this point, we've moved all debug functions to the dashboard instead of the joystick, meaning that the upper joystick buttons (the easier ones to press) are available. I suggest putting your commands up there now, to increase the chance that the drivers will try them out and acclimate to them.